docs/docs/usage.md

```diff
@@ -0,0 +1,61 @@
+
+# Getting started
+
+Add to your `project/plugin.sbt` file the following line
+
+```tut
```

Comment:
Is there a reason you're using `tut` instead of `mdoc`? If not, I would recommend switching to mdoc before you start writing a lot of docs.

`tut` and `mdoc` are similar tools, but `mdoc` is actively maintained and gaining in popularity. It's also lightning-fast if you run it with the `--watch` option, so you can see your docs update almost instantly while you write them.

Reply:
We are going to migrate to mdoc, for sure :wink: